internal/http/services/owncloud/ocdav/put.go

```diff
@@ -278,6 +278,9 @@ func (s *svc) handlePut(ctx context.Context, w http.ResponseWriter, r *http.Requ
 		httpReq.Header.Set(HeaderLockHolder, lockholder)
 	}
 
+	// Set Content-Length
```

Comment:
I wonder if we should add a few more comments here. The bug in EOS is fixed, but it's still recommended to always pass the length.